File: GILua/src/main/java/org/anime_game_servers/gi_lua/models/scene/group/SceneMonsterPool.java
package org.anime_game_servers.gi_lua.models.scene.group;

import lombok.Getter;
import org.anime_game_servers.core.base.annotations.lua.LuaNames;

@Getter
public class SceneMonsterPool {
    @LuaNames("pool_id")
    private int poolId;
    @LuaNames("rand_weight")
    private int randWeight;
}

File: GILua/src/main/java/org/anime_game_servers/gi_lua/models/scene/group/SceneMonsterPool.kt
package org.anime_game_servers.gi_lua.models.scene.group

import org.anime_game_servers.core.base.annotations.lua.LuaNames

data class SceneMonsterPool (
    @field:LuaNames("pool_id")
    val poolId: Int = 0,

    @field:LuaNames("rand_weight")
    val randWeight: Int = 0,
)
